src/main/java/com/demo/controller/BaseController.java

```diff
@@ -35,5 +38,13 @@ public String welcome(){
         log.info("welcome");
         return "welcome";
     }
+
+    @RequestMapping("/index3")
+    public String index3(Model model){
+        log.info("index");
+        int a = 1;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "a" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)